sorc/chgres_cube.fd/search_util.F90

```diff
@@ -215,27 +215,47 @@ subroutine search (field, mask, idim, jdim, tile, field_num, latitude, terrain_l
 
  end subroutine search
 
-!> Set sst values based on latitude.
+!> Set default sst values based on latitude.
 !!
-!! @param latitude latitude input
-!! @param sst sst guess value to be set
+!! Based loosely on the average annual SST
+!! values from ./fix_am/cfs_oi2sst1x1monclim19822001.grb
+!!
+!! The temperature in the polar and tropical regions
+!! is set to 273.16/300 Kelvin. Polar regions are
+!! poleward of 60 degrees. Tropical regions are within
+!! 30 degrees of the equator. In mid-latitudes, a
+!! linear change with latitude is used. 
+!!
+!! @param [in] latitude Latitude in degrees
```

Comment:
Thanks for adding units! ;-) Always a good idea!!

Is this degrees and minutes or degrees and hundredths?